clients/admin-ui/src/features/system/add-multiple-systems/AddMultipleSystems.tsx

```diff
@@ -259,13 +259,11 @@ export const AddMultipleSystems = ({ redirectRoute }: Props) => {
           totalSelectSystemsLength > 1 ? "s" : ""
         }`}
       />
-      {isFilterOpen ? (
```

Comment:
Was this not needed in the first place?

Reply:
yes and no. Sometimes you want the state to reset in the modal after it closes and this is an easy way to do that. 

I did that out of habit and it caused this bug. It's more of a case by case thing